src/components/Layout/index.tsx

```diff
@@ -252,6 +252,7 @@ const Layout: React.FC<LayoutProps> = ({
           <main id="main" className="homepage-wrapper">
             {children}
             {!homepage && <ic-back-to-top target="main" />}
+            {<ic-back-to-top target="main" />}
```

Comment:
This is the same as the line above, is it possible to just remove the !homepage from the line above instead?

Reply:
Recommended change made and working as expected, thank you